editor/src/core/shared/jsx-attributes.ts

```diff
@@ -186,6 +221,17 @@ export function jsxAttributeToValue(
   switch (attribute.type) {
     case 'ATTRIBUTE_VALUE':
       return attribute.value
+    case 'JS_IDENTIFIER':
+      return requireResult[attribute.name] ?? inScope[attribute.name]
```

Comment:
what is requireResult in our case? does it represent an import? from just reading the code here it looks to me like we want to prioritize inScope (assuming that contains all the nested scopes, including the closest scope around this identifier expression)

Reply:
Ah, yes, I think that was my intention, but not what I wrote.